dist/tagmanager-wrap.js

```diff
@@ -0,0 +1,165 @@
+(function (global, factory) {
+  if (typeof define === "function" && define.amd) {
+    define(['exports'], factory);
+  } else if (typeof exports !== "undefined") {
+    factory(exports);
+  } else {
+    var mod = {
```

Comment:
The project is being written with ES6, do not we have to change `var` to` let`?

Reply:
@williamcosta :1st_place_medal:

Reply:
@williamcosta That is the dist file, it is ok to be using var 😄

Reply:
@iondrimba I had not seen the dist file. OK.